src/bin/cargo/cli.rs

```diff
@@ -184,14 +184,13 @@ fn execute_subcommand(config: &mut Config, args: &ArgMatches<'_>) -> CliResult {
 }
 
 fn cli() -> App {
-    App::new("cargo")
+    App::new(clap::crate_name!())
```

Comment:
Why make this change? ISTM this should be the name of the application, not the name of the crate?

Reply:
Ok. I fixed it.